.changeset/good-worms-heal.md

```diff
@@ -0,0 +1,13 @@
+---
+'@astrojs/markdoc': patch
+'astro': patch
+---
+
+The function
+`emitESMImage` has been deprecated, and it will be removed in a next major release of Astro. Use the function
+`emitImageMetadata` instead.
```

Comment:
```suggestion
Deprecates the asset utility function `emitESMImage()` and adds a new `emitImageMetadata()` to be used instead

The function `emitESMImage()` is now deprecated. It will continue to function, but it is no longer recommended nor supported. This function will be completely removed in a next major release of Astro. 

Please replace it with the new function`emitImageMetadata()` as soon as you are able to do so:
```